sky_alert/endpoint.py

```diff
@@ -18,11 +19,22 @@ def healthz() -> str:
 
 
 @app.get("/openweather_sun_data")  # type: ignore
-# https://stackoverflow.com/questions/75347974/mypy-untyped-decorator-makes-function-main-untyped-for-fastapi-routes
 def sun_data(lat: str = "0", lon: str = "0") -> Union[SunData, int]:
-    return ows.get_sunrise_sunset_from_json(lat=lat, lon=lon)
+    return ows.get_sun_data(lat=lat, lon=lon)
 
 
 @app.get("/openweather_moon_data")  # type: ignore
 def moon_data(lat: str = "0", lon: str = "0") -> Union[MoonData, int]:
-    return ows.get_moonrise_moonset_from_json(lat=lat, lon=lon)
+    return ows.get_moon_data(lat=lat, lon=lon)
+
+
+@app.get("/openweather_cloud_data")  # type: ignore
+def cloud_data(lat: str = "0", lon: str = "0") -> Union[CloudData, int]:
+    return ows.get_cloud_data(lat=lat, lon=lon)
+
+
+@app.get("/get_and_write_openweather_data_to_file")  # type: ignore
+def get_and_write_openweather_data_to_file(lat: str = "0", lon: str = "0") -> None:
+    ows.update_most_recent_weather(lat=lat, lon=lon)
+    with open("most_recent_weather.json", "w") as f:
+        json.dump(ows.most_recent_weather[(lat, lon)], f)
```

Comment:
Maybe we could add something to indicate whether writing failed, ie 
```suggestion
    ows.update_most_recent_weather(lat=lat, lon=lon)
    with open("most_recent_weather.json", "w") as f:
        json.dump(ows.most_recent_weather[(lat, lon)], f)
    print("writing succeeded")
    return True
```

Reply:
something better worded